### DISCUSSION TITLE
Workflow deployment using Jenkins

### DISCUSSION THREAD

**Discussion by srp-asach** [Netflix/conductor#3388] [Q&A]
I am looking for workflow deployment using Jenknins. Could anyone please assist me here.

Thanks in advance.

**Comment by v1r3n** [comment]
Hi @srp-asach you can use the metadata APIs (`api/metadata/`) to deploy workflows and tasks.  Here is what I would o:

1. checkout workflows  and task definitions from version control repo
2. Call the metadata APIs to update the workflow and task definitions. There are separate APIs for workflows and task definitions you can cll.
